### PR TITLE
Updated design

### PR DESCRIPTION
**Pull request by astrofrog**
Here is an improvement (in my opinion) to the website. Preview: http://astrofrog.github.com/astropy-website

One thing that didn't look good before was that some pages were quite empty, and it was probably going to stay that way to avoid duplicating too much from the docs (e.g. contributing, documentation). Also, there were too many pages. This ain't the 90s... Therefore, I simplified some of the content and moved it all to the front page. I thought it looked a bit too serious before (e.g. bullet point table of contents...). So now, on the front page, you have all the information you need, which is better in itself, but incidentally means that the issue of the logo in the menubar isn't so much of a big deal because the other pages are still there are not meant for everyone, so it doesn't really matter if they are not quite as pretty. By the way, we can put the logo in the docs, so that's another matter.

Incidentally, I put back the fork me banner, and used an orange one instead, and made it go to the top of the page rather than to the bottom of the top navigation bar. I like it, but I'd be interested to hear what people think! I think it's ok to have it on the front page - in my opinion it makes it look more modern, trendy, and shows that we're open (it's not a closed effort). It's definitely not going to turn people _away_.

(cc @iguananaut, @taldcroft, @kbarbary, @eteq who were very helpful with the first iteration)

### CODE REVIEW

**Comment by eteq** [comment]
I definitely like this - it's a significant improvement (and avoids the logo argument quite adroitly :) 

One comment: There are references to the "astropy-users" group - is that a typo (supposed to be "astropy-dev"), or were you thinking of starting up that group?  (If the latter, it would be good to have a link to it somewhere on the front page - perhaps in the reporting issues section).

**Comment by eteq** [comment]
Oh, and one other thing that just occurred to me: Do we really need the search bar?  If you can tie it into the main documentation, then it's great to have there, but right now it seems like it only searches the pages that are part of the web  site, which isn't really that useful.

**Comment by embray** [comment]
I like it :)

**Comment by astrofrog** [comment]
I created the astropy-users list at the same time as astropy-dev, with the idea that it could be used for more user-based issues, but of course maybe the usual astropy (non-dev) mailing list is fine. By the way, the links to astropy-users should be on the front page - do they not work for you?

I was thinking the same about the search. I think it would be really cool to have it link to the docs, and replace 'Search' with 'Search Documentation'. I'll look into it.

**Comment by astrofrog** [comment]
@eteq - well, that was easy, the search box now points to the docs! Of course, I can't test this with the actual Astropy docs, but I tested it with a different project, and it worked. I'll update the search URL if needed once the readthedocs docs are back online.

**Comment by astrofrog** [comment]
Thinking about this again, maybe it _is_ confusing to have three mailing lists: astropy, astropy-users, and astropy-dev. Maybe we just use the standard astropy list for user questions?

**Comment by eteq** [comment]
@astrofrog - I think that's a good idea (don't use astropy-user, instead use astropy), at least for now - in the future, when we actually have some non-dev users, it might be a good idea to transition, but for now I think the best bet is to stick with the already-established astropy list.

(And the link on the front to astropy-user does work... it's just currently empty, which was why I was asking)

**Comment by astrofrog** [comment]
@eteq - I updated the links to the mailing list. I agree we should avoid fragmenting mailing lists, it's not like traffic is _that_ large on astropy at the moment.

**Comment by eteq** [comment]
Great - I definitely like the look of it now, and will report back once I get the readthdocs business straightened out.

**Comment by eteq** [comment]
readthedocs is now working again (see http://readthedocs.org/docs/astropy/en/latest/ ) - I had to add a pip requirements file with numpy.

File: _themes/bootstrap/searchbox.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {%- if pagename != "search" %}
-<form class="pull-left" action="{{ pathto('search') }}" method="get">
-  <input type="text" name="q" placeholder="Search" />
+<form class="pull-left" action="http://astropy.readthedocs.org/search.html" method="get">
```
Comment (eteq) [inline _themes/bootstrap/searchbox.html]: I think this `action` needs to be needs to be `http://astropy.readthedocs.org/en/latest/search.html`

Comment (astrofrog) [inline _themes/bootstrap/searchbox.html]: fixed!

**Comment by astrofrog** [comment]
@taldcroft and @kbarbary - any thoughts before I merge this in?

**Comment by taldcroft** [comment]
@astrofrog  - The website looks great now!  

The only issue I see is outside this pull request but about the readthedocs homepage: the logo and banner I'm seeing is HUGE (1274 x 240 px).  It probably needs a `:scale:` factor or something.  Likewise the logo in the left bar is pretty big 192 x 188 px.

**Comment by astrofrog** [comment]
@taldcroft - yes, I noticed that too. I think maybe we could go with just the logo in the left column, but keep it fairly large. It looks pretty good on other pages where the banner isn't there!

**Comment by kbarbary** [comment]
@astrofrog - Looks good.

For readthedocs, just the (fairly large) logo on the sidebar would be OK, but big logos there push the contents of the side bar pretty far down the page. Another option would be to have the "@ astropy" version of the logo at the top of the side bar instead, which would be much shorter (for a set width).

**Comment by kbarbary** [comment]
Is it possible to use a custom sphinx theme for RTD? http://python.readthedocs.org seems to do it...

**Comment by astrofrog** [comment]
@kbarbary - yes, but I have to say I kinda like the default RTD theme with the logo in the sidebar. I tried the bootstrap theme, and there are all kinds of issues, so I'd say we should stick with the RTD theme for now. It'd kind of nice to have a different theme for the docs and the front website, as it makes a clear distinction between the two. Anyway, if someone wants to change the theme, just open a PR on the main astropy repo, and we can decide.

**Comment by astrofrog** [comment]
All right, it seems there is a clear consensus - merging!

**Pull request merged**